### PR TITLE
Paper for JOSS

### PR DESCRIPTION
**Pull request by MehradSm**
+ paper.md and paper.bib have been added

### CODE REVIEW

**Comment by MehradSm** [comment]
paper.md updated regarding the referencing

**Comment by mmyros** [comment]
Nice, sure, thanks for the offer! Let me know if you need any details from me, my username on Gmail is the same as here.

**Comment by dizcza** [comment]
"_Authorship_" means that someone is the author of the package, whether it's the director of the department of the institute you're working in or the main developers. I'm not the author of *your* package. I'm a contributor. Please cite me (and other contributors) accordingly. It shouldn't be in the paper though - a list of contributors on the readthedocs is fine.

Take a look at the Elephant docs https://elephant.readthedocs.io/en/latest/ (we planned to use your package but found it convoluted with decorators and hard to follow so started developing a simplifying version). Two years ago I was the main developer of Elephant, and I'm listed as a contributor and as the author in the upcoming paper (I hope I'll be there). But all other guys - contributors - are mentioned on the readthedocs only (and of course on GitHub in a file).

My name - Danylo Ulianych - and affiliation you can take from the Elephant Authors and contributors list.

**Comment by edeno** [comment]
@mmyros could I get your affliation and orcid ID if you have one?

**Review by edeno — APPROVED** [review]

**Comment by mmyros** [comment]
> @mmyros could I get your affliation and orcid ID if you have one?

0000-0001-7790-257, National Institute of Neurological Disorders and Stroke

**Comment by edeno** [comment]
@mmyros if you could take a look at https://github.com/Eden-Kramer-Lab/spectral_connectivity/blob/master/paper.md and do a pull request for any edits that would be great. A pdf draft exists here: https://github.com/Eden-Kramer-Lab/spectral_connectivity/suites/8216191033/artifacts/358139833
